syntax = "proto3";

package screenservice;

message ScreenRequest {
  string line1 = 1;
  string line2 = 2;
  string line3 = 3;
  string line4 = 4;
  string line5 = 5;
  string returnScreen = 6;
  int32 length = 7;
  int32 showCountdown = 8;
}

message ScreenResponse {
    string status = 1;
    string message = 2;
}

service ScreenServer {
  rpc SendScreen(ScreenRequest) returns (ScreenResponse) {}
}